cms_index.php

```diff
@@ -28,19 +49,32 @@
         <textarea class="typeText" name="addAboutMeText" form="addAboutMeText"></textarea>
         <input type="submit" value="Add text">
     </form>
+    <?php
+    echo $addAboutMeSuccess;
```

Comment:
put an isset() like charlie said for aldos code review

Reply:
Done